cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/lineage/LineageDataset.java

```diff
@@ -495,12 +493,7 @@ private void addProgram(MDSKey.Builder keyBuilder, ProgramId program) {
   }
 
   private void addComponent(MDSKey.Builder keyBuilder, EntityId component) {
-    if (component instanceof FlowletId) {
-      keyBuilder.add(FLOWLET_MARKER)
-        .add(component.getEntityName());
-    } else {
-      keyBuilder.add(NONE_MARKER);
-    }
+    keyBuilder.add(NONE_MARKER);
```

Comment:
I didn't change too much of the class structure. We have to revisit this Dataset and cleanup, since there is no longer a notion of "component", in which a lot of methods are having it as parameter.